pkr_extractor/extract.c

```diff
@@ -44,7 +44,7 @@ bool ExtractDir(PKRDir *curDir){
 	uint32_t pathLen = strlen("extracted\\") + strlen((char*)curDir);
 	buffer[pathLen] = '\0';
 
-	printf("Extracting %s\n", &curDir->name);
+	printf("Extracting %s\n", curDir->name);
```

Comment:
No, removing the `&` is going backwards. It was there to give an hint to the developer that the string is inplace within the structure.

Reply:
That doesn't make any sense to me. It would have to be `&curDir->name[0]` then.

It's the difference between:
- Pointing at the first string in an array of strings.
- The first char in an array of chars (string).

If I kept the `&` I'd get warnings like this in clang:

```c
walk.c:57:31: warning: format specifies type 'char *' but the argument has type 'char (*)[32]' [-Wformat]
                printf("%s has %d files\n", &pkrDirs[i].name, pkrDirs[i].numFiles);
```